packages/react-components/README.md

```diff
@@ -14,8 +14,9 @@ other Neutrino middleware, so you can build, test, and publish multiple React co
 - Support for React Hot Loader
 - Write JSX in .js or .jsx files
 - Support for importing web workers with `.worker.*` file extensions
+- Compiles down to ES5 for all major browsers with support up to IE 9
```

Comment:
Can you change this to say "Compiles to ES5 to support major browsers, publishing to npm, and library interoperability"

Reply:
all set!